presto-docs/src/main/sphinx/release/release-0.239.rst

```diff
@@ -0,0 +1,51 @@
+* Add support for 2 security modes for views. The default `DEFINER` security mode is the same as the previous behavior. Tables referenced in the view are accessed using the permissions of the view owner (the **creator** or.
+* Add support for warning on unfiltered partition keys using `partition-keys-to-warn-on-no-filtering` system property.
+* Add support to optimize min/max only metadata query. This is controlled by existing config ``optimizer.optimize-metadata-queries`` and session property ``optimize_metadata_queries``. Note that enabling this config/session property might change query result if there are metadata that refers to empty data, e.g. empty hive partition.
+* Remove experimental feature to perform grouped execution for eligible table scans and its associated configuration property ``experimental.grouped-execution-for-elligible-table-scans`` and session property ``grouped_execution_for_eligible_table_scans``.
+* Definer** of the view) rather than the user executing the query. In the `INVOKER` security mode, tables referenced in the view are accessed using the permissions of the query user (the **invoker** of the view).
+* Enable ``dynamic-schedule-for-grouped-execution`` by default.  In future releases, we will remove this property, and grouped execution will always use dynamic scheduling.
+* Enable ``grouped-execution-for-aggregation`` and ``experimental.grouped-execution-for-eligible-table-scans`` by default.
+* Enable async page transport with non-blocking IO by default. This can be disabled by setting ``exchange.async-page-transport-enabled`` configuration property to false.
+* Introduce new configuration property ``grouped-execution-enabled`` and session property ``grouped_execution`` to turn grouped execution on or off.  This property is true by default.  If set to false, it is equivalent to setting all of ``grouped-execution-for-aggregation``, ``grouped-execution-for-join``, and ``experimental.grouped-execution-for-eligible-table-scans`` to false.  In future releases we will remove these other properties and only have a single switch for enabling and disabling grouped execution.
+* Reverting Reliable Resource Group Versioning.
+* Specify allowed roles for HTTP endpoints.
+* The worker page's thread snapshot UI does not work (no stack trace displayed on click) when there is active query load (tested under Chrome). This patch fixes an uninitialized variable in client JS that was causing this UI behavior.
+* Update JTS to 1.17.0. This changes the implementation of ST_Buffer: the output might change by a small (1e-10) amount.
+
+SPI Changes
+___________
+* Move `DistinctLimitNode` to `presto-spi` module for connectors to push down.
+
+Elasticsearch Changes
+_____________________
+* Add configurations to improve concurrency in Elasticsearch.
+* Support Elasticsearch numeric keyword.
+* Support composite publish_address in Elasticsearch.
+
+Hive Changes
+____________
+* Add support for caching the Glue metastore.
```

Comment:
We should add a release note for encryption at rest @mayankgarg1990 

* Add support for reading and writing DWRF files with encryption.  To create a table with encrypted columns....